### PR TITLE
Update docs to reflect production deployment

### LINKED ISSUE #3886: remove Heroku config
Some configuration files seem to refer to Heroku, and as far as I understand, we no longer have any Heroku dependencies. Am I wrong?

I did a `git grep` and found that we should look at:

- [ ]  `bin/post_compile`
- [ ]  a TODO note in `bin/redis-tls`
- [ ]  `config/nginx.conf.erb`
- [ ]  `vcl/main.vcl`
- [ ] `runtime.txt`
- [ ]  documentation in `docs/application.rst`

### PR DESCRIPTION
**Pull request by brainwane**
Fixes #3886.

### CODE REVIEW

File: docs/application.rst [hunk 1]
```diff
@@ -7,9 +7,13 @@ Warehouse uses the
 `Postgres <https://www.postgresql.org/docs/>`__ for its database.
 Warehouse's front end uses `Jinja2 <http://jinja.pocoo.org/>`__ templates.
 
-The production deployment for Warehouse is in progress and currently
-does not use any containers, although we may change that in the
-future. In the development environment, we use several `Docker`_  containers, and use `Docker Compose <https://docs.docker.com/compose/overview/>`__ to `manage <https://github.com/pypa/warehouse/blob/master/docker-compose.yml#L3>`__
+The production deployment for Warehouse is on Amazon Web Services (for
+PyPI.org) and Heroku (for test.pypi.org). It does not use any
+containers, although we may change that in the future. In the
```
Comment (di) [inline docs/application.rst]: The future is here! We definitely use containers in prod now. @ewdurbin might be able to phrase this best, might be worth talking about Cabotage here as well.

**Comment by brainwane** [comment]
@ewdurbin @dstufft would you review? Thanks!

**Comment by nlhkabu** [comment]
ping @ewdurbin

**Comment by ewdurbin** [comment]
superseded by #5086

**Pull request closed without merging**